src/main/java/uk/gov/hmcts/reform/judicialapi/controller/JrdUsersController.java

```diff
@@ -88,4 +92,51 @@
         return judicialUserService.fetchJudicialUsers(size, page, userRequest.getUserIds());
     }
 
+    @ApiOperation(
+            value = "This endpoint will be used for user search based on partial query. When the consumers "
+                    + "inputs any 3 characters, they will call this api to fetch "
+                    + "the required result.",
+            notes = "**Valid IDAM role is required to access this endpoint**",
```

Comment:
does this require some specific role or just valid user ? if role is required then can be mentioned.

Reply:
as per the design statement below it just needs any valid role
`User authentication will be done in the API to allow any valid IDAM role should be able to access this API `

Reply:
Generally in swagger notes we mention what role is required, but since it is mentioned "Valid IDAM role is required" thats why I raised this.